packages/cli/src/commands/dev/dev.ts

```diff
@@ -27,76 +25,59 @@ export default async function dev(
   let cwd = resolve(dir);
   const listen = parseListen(opts['--listen'] || '3000');
 
-  // retrieve dev command
-  let [link, frameworks] = await Promise.all([
-    getLinkedProject(client, cwd),
-    getFrameworks(client),
-  ]);
-
-  if (link.status === 'not_linked' && !process.env.__VERCEL_SKIP_DEV_CMD) {
```

Comment:
We need this `process.env.__VERCEL_SKIP_DEV_CMD` check still in-place for the tests. When the env var is set, we need to skip the project linking stuff.